Utilities/QC/QCMetricsExtractor/include/MetricsExtractor.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <fstream>
+#include <string>
+
+#include <boost/property_tree/ptree.hpp>
+#include <dds_intercom.h>
+

```

Comment:
Please add some documentation to both the class in general and the individual functions. AliceO2 intends to use doxygen for building the code documentation, so please use doxygen markdown from the beginning.

NB: we are from having a good automatically created code documentation, that's both because we are lacking the documentation in the code itself and a structured built.